analysis/local_analog_calibration.py

```diff
@@ -143,7 +283,7 @@ def main(lR23, lO32, OH, out_pdf, ID=[], lR23_err=[], lO32_err=[],
         ax[1].scatter(lO32[nn], OH[nn], color=ctype[nn], edgecolor='none',
                       marker=marker[nn], alpha=0.5)
         if len(ID) != 0:
-            for ii in range(len(lO32[nn])):
+            for ii in range(len(lO32[nn]-2)):
```

Comment:
Why are we excluding the last two points?

Reply:
See above